jsdoc-template/docs.html.handlebars

```diff
@@ -20,6 +20,9 @@
                 <li class="active"><a href="#">Documentation</a></li>
             </ul>
         </div>
+        <div class="navbar-right">
+          <iframe class="forkme" src="http://ghbtns.com/github-btn.html?user=ramda&repo=ramda&type=fork&size=large" height="30" width="100" frameborder="0" scrolling="0" style="width:100px; height: 30px;" allowTransparency="true"></iframe>
```

Comment:
i don't think it's necessary to encode the ampersand inside a link, except maybe in xhtml